packages/apps/plugins/plugin-stack/src/components/StackMain.tsx

```diff
@@ -79,19 +80,20 @@ const StackMainImpl = ({ sections }: { sections: StackSections }) => {
   const [_, setIter] = useState([]);
   const { t } = useTranslation('dxos:stack');
 
-  useEffect(() => {
-    // todo(thure): TypeScript seems to get the wrong return value from `ObservableArray.subscribe`
-    return sections[subscribe](() => setIter([])) as () => void;
-  }, []);
+  // todo(thure): Is there a hook that is compatible with both `ObservedArray`s and `TypedObject`s?
```

Comment:
echo objects/arrays have `subscribe` key as well, I suspect what you may have run into though is that sections is the result of an echo query, which would just be a regular array I think @dmaretskyi perhaps something for us to think about with these apis

Reply:
I see — I’d definitely prefer to have consistent interface for subscribing for both cases.